app/views/errors/maintenance.html.erb

```diff
@@ -0,0 +1,12 @@
+<header class="page-header group">
+  <div>
+    <h1>Maintenance</h1>
+  </div>
+</header>
+<div class="article-container">
+  <article role="article" class="group">
+    <div class="inner">
+      <p><%= ENV["MAINTENANCE"] %></p>
```

Comment:
Can we have a default message:

The GOV.UK Trade Traiff tool is currently unavailable due to scheduled maintenance.